bids/analysis/variables.py

```diff
@@ -609,7 +611,7 @@ def split(self, grouper):
 
         subsets = []
         for i, (name, g) in enumerate(data.groupby(grouper)):
-            name = '%s/%s' % (self.name, name)
+            name = '%s_%s' % (name, self.name)
```

Comment:
This syncs up naming conventions to the example JSON:

```JSON
    {
      "level": "session",
      "transformations": [
        {
          "name": "split",
          "input": [
            "finger_vs_others"
          ],
          "by": "session"
        }
      ]
    },
    {
      "level": "subject",
      "model": {
        "variables": [
          "test_finger_vs_others",
          "retest_finger_vs_others"
        ]
      },
      "contrasts": [
        {
          "name": "session_diff",
          "condition_list": [
            "test_finger_vs_others",
            "retest_finger_vs_others"
          ],
          "weights": [
            1,
            -1
          ],
          "type": "T"
        }
      ]
    },
```

Reply:
Cool. Do you have a preference regarding nomenclature? I haven't been very principled about what I use in different places. Nothing is decided yet for the spec, so probably best to decide what you and I like best, and then I can run it by a few others before locking things in for the spec.

Reply:
I think the `_` looks slightly better, but I could see an argument for `/` as a kind of namespace separator while `_` is just a standard character.

Reply:
I share your aesthetic preference, but also agree that `/` is less likely to occur in variable/level names--I think that was the motivation for `/`. But on reflection, `/` could conceivably cause problems in cases where file paths are involved (e.g., if someone writes a script that writes out every variable to a file, or something). Maybe `.`? That seems both uncommon in names, and less problematic from a file system perspective (I don't think confusion with file extensions is a plausible concern).

Reply:
`.` sounds good. I'll switch that before rebasing.